File: code/Person.scala
class Person(val name: String, val age: Int) {
  def showInfo(): Unit = {
    println("Name: " + name + ", Age: " + age + "...")
  }
}

object Person {
  def main(args: Array[String]): Unit = {
    val person = new Person("Gabriel", 43)
    person.showInfo()
  }
}

File: code/Person.java
public class Person {
        private String name;
        private int age;

        public Person (String name, int age) {
            this.name = name;
            this.age = age;
        }
        public void showInfo() {
            System.out.println("Name: " + name + ", Age: " + age);
        }
        public static void main(String[] args) {
            Person person = new Person("Gabriel", 43);
            person.showInfo();
        }
}


